src/test/unit/DelegationUnit.t.sol

```diff
@@ -3944,6 +3952,333 @@ contract DelegationManagerUnitTests_completeQueuedWithdrawal is DelegationManage
+        _setOperatorMagnitude(operator, strategyMock, newMagnitude);
+        cheats.prank(address(allocationManagerMock));
+        cheats.expectEmit(true, true, true, true, address(delegationManager));
+        emit OperatorSharesDecreased(operator, address(0), strategyMock, sharesToBurn);
+        cheats.expectEmit(true, true, true, true, address(delegationManager));
+        emit OperatorSharesBurned(operator, strategyMock, sharesToBurn);
+        delegationManager.decreaseAndBurnOperatorShares({
+            operator: operator,
+            strategy: strategyMock,
+            prevMaxMagnitude: WAD,
+            newMaxMagnitude: newMagnitude
+        });
+
+        // 5. Assert expected values
+        uint256 queuedSlashableSharesAfter = delegationManager.getSlashableSharesInQueue(operator, strategyMock);
+        uint256 operatorSharesAfter = delegationManager.operatorShares(operator, strategyMock);
+        assertEq(queuedSlashableSharesBefore, 0, "there should be no slashable shares in queue");
+        assertEq(queuedSlashableSharesAfter, 0, "there should be no slashable shares in queue");
+        assertEq(operatorSharesAfter, operatorSharesBefore - sharesToBurn, "operator shares should be decreased by sharesToBurn");
+    }
+
+    /**
+     * @notice Test burning shares for an operator with slashable queued withdrawals in past week.
+     * There exists a single withdrawal that is slashable.
+     */
+    function testFuzz_decreaseAndBurnOperatorShares_SingleSlashableWithdrawal(Randomness r) public {
+        // 1. Randomize operator and staker info
+        // Operator info
+        address operator = r.Address();
+        uint64 newMagnitude = 5e17;
+        // First staker
+        address staker1 = r.Address();
+        uint256 shares = r.Uint256(1, MAX_STRATEGY_SHARES);
+        // Second Staker, will queue withdraw shares
+        address staker2 = r.Address();
+        uint256 depositAmount = r.Uint256(1, MAX_STRATEGY_SHARES);
+        uint256 withdrawAmount = r.Uint256(1, depositAmount);
+
+        // 2. Register the operator, set the staker deposits, and delegate the 2 stakers to them
+        _registerOperatorWithBaseDetails(operator);
+        {
+            // Set the first staker deposits in the strategies
+            IStrategy[] memory strategyArray = strategyMock.toArray();
+            uint256[] memory sharesArray = shares.toArrayU256();
+            uint256[] memory depositArray = depositAmount.toArrayU256();
+            strategyManagerMock.setDeposits(staker1, strategyArray, sharesArray);
+            // Set the second staker's deposits in the strategies
+            strategyManagerMock.setDeposits(staker2, strategyArray, depositArray);
+        }
+        _delegateToOperatorWhoAcceptsAllStakers(staker1, operator);
+        _delegateToOperatorWhoAcceptsAllStakers(staker2, operator);
+
+        // 3. Queue withdrawal for staker2 and roll blocks forward so that the withdrawal is not slashable
```

Comment:
Nit: typo, test not rolling blocks forward

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/881/commits/4799402ad08aacd77cf121280bb2e91e62e5dfe8